modules/programs/starship.nix

```
@@ -89,7 +89,7 @@ in {
       mkIf (cfg.settings != { }) { source = configFile cfg.settings; };
 
     programs.bash.initExtra = mkIf cfg.enableBashIntegration ''
-      if [[ -z $INSIDE_EMACS ]]; then
+      if [[ -z $INSIDE_EMACS ]] && [[ $TERM != "dumb" ]]; then
```

I think we could use the [same as NixOS](https://github.com/marsam/nixpkgs/blob/f17f830897fcfc75914bd4a7d4ccd2cdd5485f81/nixos/modules/programs/bash/bash.nix#L106):
```suggestion
      if [ "$TERM" != "dumb" -o -n "$INSIDE_EMACS" ]; then
```

---

I actually think it was correct as-is. We want to run the command if `INSIDE_EMACS` is empty and the terminal is non-dumb. With the new expression starship is run if `INSIDE_EMACS` is non-empty or the terminal is non-dumb.